config/prod/TESLA-download-bucket.yaml

```diff
@@ -4,14 +4,14 @@ template_path: "remote/s3-bucket-v2.j2"
 stack_name: "TESLA-download-bucket"
 sceptre_user_data:
   # (Optional) Synapse IDs for owner.txt file
-  SynapseIDs: ["1968150", "3382314"]  # Kristen Dang and Xengie Doan
+  SynapseIDs: ["3382314"]  # Xengie Doan
 parameters:
   # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
   Department: "CompOnc"
   # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
   Project: "TESLA"
   # The resource owner
-  OwnerEmail: "kristen.dang@sagebase.org"
+  OwnerEmail: "xengie.doan@sagebase.org"
```

Comment:
who should be owner now @xdoan ?

Reply:
@xdoan for now, then probably @jaeddy

Reply:
Since Xengie leaves tomorrow, shouldn't this be someone else?